### DISCUSSION TITLE
Keyboard lockout / Freeze during Moodle quiz activity in SEB 3.1.1 (Moodle 4.1)

### DISCUSSION THREAD

**Discussion by spicymoodles** [SafeExamBrowser/seb-win-refactoring#844] [Q&A]
Moodle 4.1
SEB 3.1.1.250
Win 10 Enterprise

Hi, 

I'll start by saying we have a solution to the below (upgrading SEB) but very curious to know what went wrong as we have experienced quite many issues with SEB over the years. 

We've had a few incidents where switching between questions in Moodle's 'Quiz navigation' menu causes the keyboard to stop functioning. The issue only happens while using SEB. Attempting the quiz outside of SEB has no issues. 

ie: If I click on Q8 then jump back to Q1 or Q2 or other 

![Q menu RHS](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/126999106/0c6acb03-7153-46e6-bb79-807f345df872)

Sometimes it works as intended, sometimes it won't load:

![Page will not fully load](https://github.com/SafeExamBrowser/seb-win-refactoring/assets/126999106/c57cfd46-ed2b-4582-a7c3-bdb42024de4f)

The blue progress bar starts progressing but then gets stuck mid way > clicking on the 'reload' button loads the page fine > However the keyboard no longer works. (We discovered that by toggling the 'up/down' keys on the keyboard a few times the keyboard come back to life)  

- Tested a few quizzes with different types of questions (Assey, Multiple, MCQ etc;) made no difference - all break

- FIX is using latest SEB version 3.7 no issues found.
I've compared SEB logs from 3.7 and 3.1.1 with time stamps where the page reload breaks the keyboard but nothing stands out. 
The only thing that has changed is that we have upgraded Moodle from 3.9 to 4.1 this year. Is it Moodle or can it be Windows updates? TIA for any insight.

**Comment by dbuechel** [comment]
Thanks for the report. That is hard to tell and unfortunately we do not have the necessary resources to provide support for older versions, thus we'll have to leave this open; especially as you do have a working solution (which is furthermore our general recommendation), namely updating to the latest release version of SEB.
